package/libnsh/libnsh.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# libnsh
+#
+################################################################################
+
+LIBNSH_VERSION = 0.4
+LIBNSH_PKGNAME = libnsh
+LIBNSH_SOURCE = $(LIBNSH_PKGNAME)-$(LIBNSH_VERSION).tar.gz
+# This looks like a typo, but the 'v' needs to be there
```

Comment:
This is a common thing, check with other packages in Buildroot, you do not need to mention it here as a comment.  If you like, it can be included in the commit message.